mkdocs/build.py

```diff
@@ -149,7 +150,7 @@ def build_sitemap(config, env, site_navigation):
 
 def build_template(template_name, env, config, site_navigation=None):
 
-    log.debug("Building {0} page".format(template_name))
+    log.debug("Building %s page", template_name)
```

Comment:
Yeah, technically speaking this is the correct way to use the logging API. It should be passed in unformatted and interpolation left to the logging function.